CHANGELOG.md

```diff
@@ -7,9 +7,12 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 ### Bug Fixes:
 - CI: use ubuntu-18.04 for gcc-5 and gcc-6 builds #1252
 - fix OpenMP 5 shared memory allocation #1254
+- fix CPU static shared memory implementation #1258
+- units tests: fix queue test #1266
```

Comment:
Typo: should be `unit`, not `units`. Not sure if a big deal, but in case it is a public changelog worth fixing I think.

Reply:
Forgot to ping @psychocoderHPC

Reply:
Well spotted, I will fix it. Therefore we have the review process. No need to ping, the PR author will by default be notified about any comments.

Reply:
fixed